doc/cephadm/operations.rst

```diff
@@ -305,3 +305,46 @@ the cluster, create an initial ``ceph.conf`` file. For example::
 Then, run bootstrap referencing this file::
 
   cephadm bootstrap -c /root/ceph.conf ...
+
+
+.. _cephadm-removing-hosts:
+
+Removing Hosts
+==============
+
+If the node that you are going to remove runs OSDs, remove the OSDs from it 
+first and check that no OSDs are running on that node.
+
+To remove a node from a cluster, do the following:
+
+1. For all Ceph service types except for ``node-exporter`` and ``crash``, remove 
+  the node from the placement specification file (for example, cluster.yml). 
+  For example, if you are removing the host named host2, remove all occurrences of 
+  ``- host2`` from all ``placement:`` sections.
+
+  Update::
```

Comment:
```suggestion
  Update
  .. code-block:: yaml
  
    service_type: rgw
    # ...
```

Reply:
@sebastian-philipp the comment part was just to avoid repeating your sample. it's not supposed to be included in your change. i think you just incorporated my suggestion as it is...

Reply:
I was super distracted today. I'll have a look tomorrow. ok?

Reply:
@sebastian-philipp sure. sorry for the confusion. thanks!